test/connection.test.ts

```diff
@@ -2077,6 +2077,17 @@ describe('Connection', () => {
     );
   });
 
+  it('get blocks', async () => {
+    await mockRpcResponse({
+      method: 'getBlocks',
+      params: [0, 1],
+      value: [1, 2, 3, 4, 5],
```

Comment:
I would prefer if this value is realistic.. we shouldn't return 5 blocks for a 2 slot range

Reply:
I have put something similar to that one: https://docs.solana.com/developing/clients/jsonrpc-api#getblocks

However, I can replace it with current real values:

```
curl https://solana-api.projectserum.com -X POST -H "Content-Type: application/json" -d '
  {"jsonrpc": "2.0","id":1,"method":"getBlocks","params":[96460288, 96460330]}
'
{"jsonrpc":"2.0","result":[96460288,96460289,96460290,96460291,96460292,96460293,96460294,96460295,96460296,96460297,96460298,96460299,96460300,96460301,96460302,96460303,96460304,96460305,96460306,96460307,96460308,96460309,96460310,96460311,96460316,96460317,96460318,96460319,96460320,96460321,96460322,96460323,96460324,96460325,96460326,96460327,96460328,96460329,96460330],"id":1}
```

Reply:
> I have put something similar to that one: https://docs.solana.com/developing/clients/jsonrpc-api#getblocks

Yeah 5-10 sounds great!